tsfresh/feature_selection/feature_selector.py

```diff
@@ -115,8 +115,9 @@ def check_fs_sig_bh(X, y, settings=None):
     pool = Pool(settings.n_processes)
 
     # Helper function which wrapps the _calculate_p_value with many arguments already set
-    f = partial(_calculate_p_value, X=X, y=y, settings=settings, target_is_binary=target_is_binary)
-    p_values_of_features = pd.DataFrame(pool.map(f, df_features['Feature']))
+    f = partial(_calculate_p_value, y=y, settings=settings, target_is_binary=target_is_binary)
+    results = pool.map(f, [X[column] for column in set(X.columns)], chunksize=settings.chunksize)
```

Comment:
Is there a reason, why you have changes df_feature["Features"] to set(X.columns)? I know that both are the same, but I think df_features["Features"] is a bit more readable.